ui.tests/test-module/libs/support/commands.js

```diff
@@ -392,3 +392,15 @@ Cypress.Commands.add(
         return subject;
     }
 );
+
+
+// cypress command to click ? and toggle description and tooltip
+Cypress.Commands.add("toggleDescriptionTooltip", (bemBlock, fieldId) => {
+    cy.get(`#${fieldId}`).find(`.${bemBlock}__questionmark`).click();
```

Comment:
we should also check if tooltip is visible before clicking on "?" icon.

Reply:
we should also compare the description texts in the tests to validate if the correct text is shown in markup.